artifactory/utils/docker/buildinfo.go

```diff
@@ -82,7 +91,7 @@ func (builder *buildInfoBuilder) Build(module string) (*buildinfo.BuildInfo, err
 
 // Search, validate and create artifacts and dependencies of docker image.
 func (builder *buildInfoBuilder) updateArtifactsAndDependencies() error {
-	// Search for all the image layer to get the local path inside Artifactory (supporting virtual repos).
+	// Search for all the image layer to get the local path inside Artifactory (supporting virtual & remote repositories).
```

Comment:
// Search for all the image layer to get the local path inside Artifactory (supporting virtual & remote repositories).
-->
// Search for all image layers to get the local path inside Artifactory.

The following is incorrect, and I therefore removed it (following your change, all repo types are supported, not just remote and virtual):
...(supporting virtual & remote repositories).